frontend/src/index.pug

```diff
@@ -170,10 +170,15 @@ html
                   title="click to view author's repo"
                 )
                   i.summary-chart__title--button.fab.fa-github
-
+                a(
+                  title="click to view breakdown of commits",
+                  onclick="deactivateAllOverlays()",
```

Comment:
just to check, why is this needed?

Reply:
the `deactivateAllOverlays()` is to remove the any previous existing dotted lines in the ramp charts since this zooms into the entire ramp chart.